### DISCUSSION TITLE
Error for Submodule: incomeStatementHistory

### DISCUSSION THREAD

**Discussion by chiahao-codes** [gadicc/yahoo-finance2#750] [Q&A] [answered]
I am getting the following error when trying to access submodule: incomeStatementHistory. 
I'm not sure why. Has anyone else gotten this?
Any help would be much appreciated.
```
errors: [
    {
      keyword: 'yahooFinanceType',
      message: "Got {}->null for 'number', did you want 'number | null' ?",
      params: [Object],
      instancePath: '/incomeStatementHistory/incomeStatementHistory/0/totalOtherIncomeExpenseNet',
      schemaPath: '#/definitions/IncomeStatementHistoryElement/properties/totalOtherIncomeExpenseNet/yahooFinanceType',
      data: {}
    },
    {
      keyword: 'yahooFinanceType',
      message: "Got {}->null for 'number', did you want 'number | null' ?",
      params: [Object],
      instancePath: '/incomeStatementHistory/incomeStatementHistory/1/totalOtherIncomeExpenseNet',
      schemaPath: '#/definitions/IncomeStatementHistoryElement/properties/totalOtherIncomeExpenseNet/yahooFinanceType',
      data: {}
    },
```

**Comment by gadicc** [comment, marked as answer]
Hey @chiahao-codes, can you let us know which symbol / ticker you're experiencing this with?  And also, what version of the library you're using?

Lastly, Yahoo have unfortunately removed the majority of information we used to get back from `incomeStatements` in `quoteSummary`, but we can now get it instead via [fundamentalsTimeSeries](https://github.com/gadicc/node-yahoo-finance2/blob/devel/docs/modules/fundamentalsTimeSeries.md).  Just note that this module is very new with more features and a small breaking change currently in development over at https://github.com/gadicc/node-yahoo-finance2/pull/753.

**Reply by chiahao-codes** [reply]
Hi,

I was able to use fundamentalsTimeSeries module. Thank you!

**Reply by gadicc** [reply]
Great news!  Thanks for reporting back :pray: 

Also note, we just merged https://github.com/gadicc/node-yahoo-finance2/pull/753 and published it in v2.11.0.  More details in the PR but there's a small breaking change that now the specific `module` needs to specified.
